openapi/openapi.yaml

```diff
@@ -306,6 +326,25 @@ components:
         pattern: '^([0-9]{10})$'
         example: '1000477501'
       description: The ten character code compromised of a one character borough, five character block, and four character lot codes.
+    limitParam:
+      name: limit
+      required: false
+      in: query
+      schema:
+        type: string
+        pattern: '^[0-9]+$'
+        example: '100'
+      description: The maximum number of results to be returned in each response. The default limit is 20 and it cannot exceed 100.
```

Comment:
Are you sure about this logic? I think it's generally best to have APIs "fail loudly" rather change passed values in ways that consumers of the API might not expect.

Reply:
I agree that on principle it's good to be loud when things fail. In cases where the query parameter isn't even a number, we do fail loudly. 

But when the user has a value that is too high or too low, we know what they want- and we can partially fulfill their request. So in this instance, we "clearly renegotiate"- outlining the behavior in the docs and including the used limit in the response.

Reply:
Will change to throw an error